### ISSUE TITLE
[IE11 + Jaws2018] aria-describedby not getting read when html element already has "title" attribute.

### ISSUE THREAD

**Issue by ShardulGhusey** [FreedomScientific/VFO-standards-support#20]
### Summary

> aria-describedby text not getting read when html element has "title" attribute also.

### Expected result
>The aria-describedby text should be read irrespective of any other attribute on it.

### Actual result
>The aria-describedby text is not getting read when the same html element has "title" attribute in it.

### Example
[https://gist.github.com/ShardulGhusey/b9fd9f55e2f1f6590f0a74854b1be3e2](url)

### Additional Information
>aria-describedby text not getting read when "title" is present in the element, once we remove the title tag the aria-describedbby text is read correctly.

#### JAWS version and build number
Jaws 2018
 
#### Operating System and version
Windows 10

#### Browser and version:
IE 11

**Comment by stevefaulkner** [comment]
@ShardulGhusey this issue has a similar origin to #18 This is not a JAWS issue its a bug in IE. For `aria-describedby` to be exposed correctly as the accessible description for the element, the referenced element needs to be an accessible element (in IE terms). So if you add `tabindex="-1"` to the element:

```
<button tile="titel text" id="describedButton"  aria-label="This is the aria-label text " aria-describedby="descriptionId">

<div id="descriptionId" tabindex="-1">This is the description </div>
```
The text content of the element is exposed (via MSAA) as the accessible description, otherwise the `title` attribute is used as the accessible description. 

Refer to  [Practical Support: aria-label, aria-labelledby and aria-describedby](https://w3c.github.io/using-aria/#label-support)

**Comment by karlgroves** [comment]
@stevefaulkner Can you clarify why it won't be fixed in IE11? Is there documentation from Microsoft as to why?   Although IE11 is quite old, in terms of browser life, it is still being maintained.  It'd be good to have a reference to point people to if they encounter this.

Also, do you know whether this bug exists in Edge?

**Comment by patrickhlauke** [comment]
> Although IE11 is quite old, in terms of browser life, it is still being maintained

it is not. any bug reported to MS for IE11 got closed for the last few years unless security related (see for instance the comment on https://connect.microsoft.com/IE/feedback/details/2445370/ie-and-edge-title-attribute-tooltips-shown-for-keyboard-users-store-title-value-show-old-value-even-if-title-gets-changed-removed-on-focus). And the new bug tracker doesn't accept IE11 bugs at all anymore (see for instance https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/7409348/#comment-0)

**Comment by karlgroves** [comment]
Apologies. I assumed that since it was still getting updates that meant "maintained"
https://support.microsoft.com/en-us/help/4040685/cumulative-security-update-for-internet-explorer

But, admittedly, security updates still !== "maintained".

At any rate, the follow-up question stands: Does this bug also exist in Edge?

**Comment by stevefaulkner** [comment]
@karlgroves AFAIK non security related fixes stopped on IE11, quite a while back.  Also I understand that while `aria-describedby` support is constrained via MSAA, it is fully supported in UIA in IE11, need to look into what the technical practicalities of AT's using UIA in IE11 with Windows 10 are.

**Comment by stevefaulkner** [comment]
> At any rate, the follow-up question stands: Does this bug also exist in Edge?
As this is an issue particular to MSAA, and MSAA is not used in Edge, I would suggest not.

**Comment by stevefaulkner** [comment]
Closing this issue as wontfix as its IE only
